src/routes/components/FormItemPage.tsx

```diff
@@ -173,23 +184,111 @@ export default function FormItemPage() {
                 onChange(name: string, value: string) {
                   setValue(value);
                 }
-              `}/>
+              `}
+              />
 
               <GoAFormItem {...formItemProps}>
                 <GoAInput onChange={noop} value="" name="item" />
               </GoAFormItem>
             </Sandbox>
 
             <ComponentProperties properties={componentProps} />
+
+            <h2 id="component-examples" className="hidden" aria-hidden="true">Examples</h2>
+
+            <h3 id="component-example-sortable-columns">Slotted Helper Text</h3>
+            <Sandbox skipRender>
+              <GoAFormItem 
+                label="First name" 
+                helpText={helptextReactNode}>
+                <GoAInput onChange={noop} value="" name="item" />
```

Comment:
Try using the following style
```
<GoAFormItem 
    label="First name" 
    helpText={helptextReactNode}
>
    <GoAInput onChange={noop} value="" name="item" />
</GoAFormItem>
```

You will also need to do this in all the other instances as well.

Reply:
Formatted the entire file using Prettier. 
Also, the code snippets are formatted manually.